infra/scripts/helm/validate-helm-chart-versions.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 # Amount of file locations that need to be bumped in unison when versions increment
-UNIQUE_VERSIONS_COUNT=18
+UNIQUE_VERSIONS_COUNT=20
```

Comment:
why is this incrementing by 2 given that we're adding 3 new files to `files_to_bump.txt`?

Reply:
this is just for helm charts, so that'd be two files